pid_param:
    p_x:
        p : 0.6
        i : 0.0
        d : 0.0
        max_i: 40
    p_y:
        p : 0.6
        i : 0.0
        d : 0.0
        max_i: 40
    v_x:
        p : 0.7
        i : 0.05
        d : 0.0
        max_i: 30

    v_y:
        p : 0.7
        i : 0.05
        d : 0.0
        max_i: 30

    p_z:
        p : 1.0
        i : 0.02
        d : 0.02
        max_i : 10
        
    v_z:
        p : 0.5
        i : 0.1
        d : 0.1
        max_i: 1000

    thr:
        p : 0.01
        i : 0.006
        d : 0.000
        max_i: 30.0
        level_thrust: 0.5
